_layouts/home.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: home
+---
+
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: "en" }}">
+
+{%- include head.html -%}
+
+  <body class="home">
+    <main class="home__content" aria-label="Content">
+        <div class="home__left">
+          <div class="home__left-inner">
+            <img class="home__title" src="{{relativeBase}}assets/images/lockbox-logo.svg" alt="{{ site.title | escape }}">
+            <h1>{{ content }}</h1>
+          </div>
+        <div class="home__app-buttons">
+          <a class="app-store" href="{{ site.ios_link | escape }}">
+            <img src="{{relativeBase}}assets/images/app-store.svg" alt="Apple App Store" height=" 50px" width="auto">
```

Comment:
I don't think the `height=` tag likes "px" suffixes (that's only a CSS thing IIRC). Probably smart enough to ignore it, but...

Probably same w/ `width="auto"`.

Once this is hosted somewhere, I can run Google Lighthouse and aXe and webhint.io scanners though and check for other potential a11y issues.